extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js

```diff
@@ -159,6 +176,52 @@ export class AmpStoryInteractiveSlider extends AmpStoryInteractive {
       (event) => event.stopPropagation(),
       true
     );
+    this.storeService_.subscribe(
+      StateProperty.CURRENT_PAGE_ID,
+      (currPageId) => {
+        const isPostState =
+          this.rootEl_.classList.contains(POST_SELECTION_CLASS);
+        if (isPostState) {
+          // If it's already been interacted with, do not animate.
+          return;
+        }
+        if (currPageId != this.getPageEl().getAttribute('id')) {
+          // Resets animation when navigating away.
+          cancelAnimationFrame(this.currentRAF_);
+          clearTimeout(this.landingAnimationDelayTimeout_);
+          this.inputEl_.value = 0;
+          this.onDrag_();
+          this.rootEl_.classList.remove(MID_SELECTION_CLASS);
+          return;
+        }
+        let startTime;
+        const animateFrame = (currTime) => {
+          // Set current startTime if not defined.
+          if (!startTime) {
+            startTime = currTime;
+          }
+          const elapsed = currTime - startTime;
+          if (HINT_ANIMATION_DURATION_MS < elapsed) {
+            this.rootEl_.classList.remove(MID_SELECTION_CLASS);
+            return;
+          }
+          // Value between 0 and 1;
+          const timePercentage = elapsed / HINT_ANIMATION_DURATION_MS;
+          const val =
+            timePercentage < 0.5
+              ? easeInOutCubic(timePercentage * 2) * 30
+              : easeInOutCubic(2 - timePercentage * 2) * 30;
+          this.inputEl_.value = val;
+          this.onDrag_();
+          this.currentRAF_ = requestAnimationFrame(animateFrame);
+        };
+        this.landingAnimationDelayTimeout_ = setTimeout(
+          () => requestAnimationFrame(animateFrame),
```

Comment:
@processprocess don't we need to set `this.currentRAF_` to this `requestAnimationFrame`?